libs/vkd3d/vkd3d_private.h

```diff
@@ -734,6 +734,7 @@ enum vkd3d_resource_flag
     VKD3D_RESOURCE_LINEAR_TILING          = (1u << 4),
     VKD3D_RESOURCE_EXTERNAL               = (1u << 5),
     VKD3D_RESOURCE_ACCELERATION_STRUCTURE = (1u << 6),
+    VKD3D_RESOURCE_SIMULTANEOUS_ACCESS    = (1u << 7)
```

Comment:
`,` at the end, unless we want to change two lines next time we add a flag.

Reply:
Fixed. pretty sure trailing `,` is not really compliant invalid, but w/e, we do that everywhere.

Reply:
It's actually explicitly allowed in C99.

Reply:
I must be confusing this with C++